script/ArbitrumMainnetDeployer.sol

```diff
@@ -15,55 +17,76 @@ contract ArbitrumMainnetDeployer is BaseDeployer {
 
     address constant sequencerFeed = address(0xFdB631F5EE196F0ed6FAa767959853A9F217697D);
 
-    constructor() {
-        chainId = 42_161;
+    function defaultHubParams() internal pure returns (BaseDeployer.HubParams memory) {
+        return BaseDeployer.HubParams({
+            minDuration: 30 days,
+            maxDuration: 365 days,
+            minLTV: 2500,
+            maxLTV: 9000
+        });
     }
 
-    function defaultHubParams() internal pure override returns (HubParams memory) {
-        return HubParams({ minDuration: 30 days, maxDuration: 365 days, minLTV: 2500, maxLTV: 9000 });
-    }
+    function deployAndSetupFullProtocol(address owner)
```

Comment:
This is the one part that needs to be duplicated between the two deployers (because there's no inheritance with libraries). 

It's pretty minimal so should be easy to compare.